docs/nddata/mixins/ndslicing.rst

```diff
@@ -89,23 +85,35 @@ printed and the property will be kept as is::
     >>> data = np.array([1, 2, 3, 4])
     >>> mask = False
     >>> uncertainty = StdDevUncertainty(0)
-    >>> wcs = {'a': 5}
-    >>> ndd = NDDataRef(data, mask=mask, uncertainty=uncertainty, wcs=wcs)
+    >>> ndd = NDDataRef(data, mask=mask, uncertainty=uncertainty)
     >>> ndd_sliced = ndd[1:3]
     INFO: uncertainty cannot be sliced. [astropy.nddata.mixins.ndslicing]
     INFO: mask cannot be sliced. [astropy.nddata.mixins.ndslicing]
-    INFO: wcs cannot be sliced. [astropy.nddata.mixins.ndslicing]
 
     >>> ndd_sliced.mask
     False
 
+
+Slicing NDData with World Coordinates
+-------------------------------------
+
+If ``wcs`` is set, it must be either implement
+`~astropy.wcs.wcsapi.BaseLowLevelWCS` or `~astropy.wcs.wcsapi.BaseHighLevelWCS`.
+This means that only integer or range slices without a step are supported. So
+slices like ``[::10]`` or array or boolean based slices will not work.
+
+If you want to slice an ``NDData`` object called ``ndd`` without the WCS you can remove the
+WCS from the ``NDData`` object by running:
+
+    >>> del(ndd.wcs)
```

Comment:
This seems unnecessary and hacky? Why support this?

Reply:
Hacky in what regard? I am happy to go with consensus on this, I only added it to provide an easy way of detaching the WCS from the NDData if you want to slice the remaining arrays with something WCS can't support.

Reply:
Does `ndd.wcs = None` also work?

Reply:
I think it's just the ``del()`` that felt hacky, I think setting to None seems more Pythonic

Reply:
It doesn't work, there is no setter on the WCS property. I went with this because it limited you to just removing the WCS rather than setting it to anything else (which seemed weird to me).

Reply:
Nothing interesting to add here (setting to None and using `del` are both valid ways imo), but I wanted to say that `del` is a statement, not a function, so no need for parenthesis :slightly_smiling_face: